common/src/main/java/org/opensearch/ml/common/transport/agent/MLSearchAgentAction.java

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.ml.common.transport.agent;
+
+import org.opensearch.action.ActionType;
+import org.opensearch.action.search.SearchResponse;
+
+public class MLSearchAgentAction extends ActionType<SearchResponse> {
+    // External Action which used for public facing RestAPIs.
```

Comment:
This comment is misleading, this action is for transport layer instead of rest layer.